ONSdigital/dp-frontend-filter-dataset-controller
# Remove logs introduced only for performance tests

### What

Removed logs that were introduced for performance tests

### How to review

- Make sure only the performance tests logs have been removed. You can validate this by doing a diff against the last commit before the logs were introduced:
```
git diff e9ef7e7ca0997eb3ba57884c5e57c0855d55e98
```
  - Check that the only difference with that commit is the audit nancy fix in Makefile.
- Make sure unit tests pass

### Who can review

Anyone
